assets/benchmarks/shaders/Benchmark_RandomColour.hlsl

```diff
@@ -0,0 +1,40 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+struct RandomParams
+{
+    uint32_t Seed;
+};
+
+#if defined(__spirv__)
+[[vk::push_constant]]
+#endif
+ConstantBuffer<RandomParams> Random : register(b0);
```

Comment:
sorry where do you set this seed? more general question, is there a reason you want to output a random solid color instead of a fixed one? maybe better with a solid color that is chosen from UI (we could use a dropdown list to select some predefined colors) would be more interesting since we could see the impact of colors on the compression rate?

Reply:
Seconding this. A `return MyConstantBuffer.color;` should work fine no?

Reply:
OK, I changed this to have a dropdown list of solid colours.

Reply:
thanks :)